docs/Tutorial.md

```diff
@@ -37,6 +37,11 @@ If you want to compile your model into a code for, e.g., a specific model of NPU
 
 Backend extension should be installed _after_ installing ONE-vscode extension.
 
+### Use ONE-vscode without any backend extension
+ONE-vscode supports `ONE` backend by default. This requires [Docker Installation](https://docs.docker.com/desktop/install/ubuntu/) and [root privileges](https://docs.docker.com/engine/install/linux-postinstall/). However it supports only basic compilation. If you want to compile for a specific hardware model, please install backend extensions.
+
+![image](https://user-images.githubusercontent.com/77388214/193172812-48c79aa5-1b21-4245-bb28-95a1e8d06375.png)
```

Comment:
For consistency, could you please add the image file to this pr and use \<img> tag? (AFAIK, we used \<img> tag to avoid each images have different width. 

@Samsung/one-vscode Did we made a discussion about managing image files in documents?

Reply:
Thanks for your review! We're going to attach the image file and change the image to <img> tag. 
And is 1000 width okay?

Reply:
@lettersto 

Could you upload the image to the `docs` directory? And please use a link. :)

@Samsung/one-vscode We  have uploaded the images related to document under the `doc` directory.
However, it would be good to collect resources under the name `images` or `resources` below `docs` directory.

Reply:
Q. It's out of my curiosity, does it work now? Or after the full implementation of ONEToolchain.ts?